lib/stages/FEngine.cpp

```diff
@@ -491,9 +491,11 @@ void FEngine::main_thread() {
         } // if !skip_julia
 
         // Set metadata
-        chordMetadata* const E_metadata = get_chord_metadata(E_buffer, E_frame_id);
+        std::shared_ptr<chordMetadata> const E_metadata = get_chord_metadata(E_buffer, E_frame_id);
         chord_metadata_init(E_metadata);
-        E_metadata->chime.fpga_seq_num = frame_index;
+        // ???
+        E_metadata->frame_counter = frame_index;
```

Comment:
Do we still have a frame counter? That doesn't make sense in a ringbuffer?

Reply:
This is an FPGA-level frame counter... this may not make sense any more when the data go through the ringbuffer.  We could copy the first frame counter, or zero it out...